bin/ncp-update

```diff
@@ -9,7 +9,7 @@
   [[ "$BRANCH" != "master" ]] && echo "INFO: updating to development branch '$BRANCH'"
 
   TMPDIR="$( mktemp -d /tmp/ncp-update.XXXXXX || ( echo "Failed to create temp dir. Exiting" >&2; exit 1 ) )"
-  trap "cd /; rm -rf \"${TMPDIR}\"; exit 0" 0 1 2 3 15
+  trap "cd /; rm -rf \"${TMPDIR}\"" EXIT
```

Comment:
I think that this will do

```
  trap 'cd /; rm -rf "${TMPDIR}"; exit $?' EXIT
```

Reply:
Wouldn't that use the exit code of the rm command?

Reply:
bruh yes, look again

Reply:
Actually that would do the same as my code snippet, iirc. I looked had something like you are proposing at first, but discovered, that it was unnecessary to explicitly add the exit code inside the trap (because the outer exit code would be passed through by default).

But I'll double check to be sure

Reply:
I tested it, and it works. The traps passes on the exit code that triggered it by default (if not overwritten)